frontend/website/src/Style.re

```diff
@@ -8,6 +8,7 @@ module Colors = {
       Printf.sprintf("hsla(%d,%d%%,%d%%,%f)", h, s, l, a);
 
   let fadedBlue = `rgb((111, 167, 197));
+  let lightBlue = `rgba((71, 137, 196, 0.1));
```

Comment:
can you update to follow the other colors, eg. have a function that takes an alpha